edx_proctoring/static/proctoring/js/models/proctored_exam_model.js

```diff
@@ -0,0 +1,52 @@
+(function(Backbone) {
+    var ProctoredExamModel = Backbone.Model.extend({
+        /* we should probably pull this from a data attribute on the HTML */
```

Comment:
Is this a TODO?  Are you going to address it prior to the initial release?  If not I'd log a 'technical debt' bug and update the comment here, accordingly